config/environments/development.rb

```diff
@@ -1,3 +1,5 @@
+HOST = 'localhost:3000'
```

Comment:
I would move this inside the configuration block, although I have no idea how this should be handled as most Rails-y as possible.